GameOfRunes/GameOfRunes/GameEngine/Systems/SpriteSystem.swift

```diff
@@ -18,12 +18,18 @@ class SpriteSystem: GKComponentSystem<SpriteComponent>, System {
     
     override func addComponent(foundIn entity: GKEntity) {
         super.addComponent(foundIn: entity)
-        
+
         guard let spriteComponent = entity.component(ofType: SpriteComponent.self) else {
             return
         }
-        
-        gameEngine?.gameScene?.addNodeToLayer(layer: spriteComponent.layerType, node: spriteComponent.node)
+
+        guard let renderNode = gameEngine?.rootRenderNode else {
```

Comment:
can we change this back if we don't need to debug this

Reply:
Wait sorry I'm not sure what you mean by this haha

Reply:
like remove this guard statement/ combine it w the one above, since we don't need the print

Reply:
Done!